wishlist page/wishlist.css

```diff
@@ -0,0 +1,7 @@
+.wishlist-container{
+    width: 80%;
+    display: flex;
+    height: auto;
+    flex-wrap: wrap;
```

Comment:
You'd be using display:flex and flex-wrap: wrap in many projects, you can consider making utility classes for these.

Reply:
yes you are right will make another branch to fix this one